rtv/content.py

```diff
@@ -352,3 +366,46 @@ def get(self, index, n_cols=70):
         data['offset'] = 0
 
         return data
+
+class SubscriptionContent(BaseContent):
+    def __init__(self, subscriptions, loader):
+        self.name = "Subscriptions"
+        self._loader = loader
+        self._subscriptions = subscriptions
+        self._subscription_data = []
+
+    @classmethod
+    def get_list(cls, reddit, loader):
+        try:
+            with loader():
+                subscriptions = reddit.get_my_subreddits(limit=None)
+        except praw.errors.APIException:
+            raise SubscriptionError()
+
+        return cls(subscriptions, loader)
+
+    def get(self, index, n_cols=70):
+        """
+        Grab the `i`th subscription, with the title field formatted to fit inside
+        of a window of width `n_cols`
+        """
+
+        if index < 0:
+            raise IndexError
+
+        while index >= len(self._subscription_data):
+            try:
+                with self._loader():
+                    subscription = next(self._subscriptions)
+            except StopIteration:
+                raise IndexError
+            else:
+                data = self.strip_praw_subscription(subscription)
+                self._subscription_data.append(data)
+
+        data = self._subscription_data[index]
+        data['split_title'] = wrap_text(data['name'], width=n_cols)
```

Comment:
So I think in this case, ``subreddit.title`` (data['title']) is actually the one that should be wrapped and not the name. The title is the property with the subreddit description and may be longer than the width of the terminal.